pages/en/lb2/datasources.json.md

```diff
@@ -80,17 +80,17 @@ To connect a model to a data source, follow these steps:
     ```shell
     $ apic create --type datasource
     ? Enter the data-source name: mysql-corp
-    ? Select the connector for mysql: MySQL (supported by StrongLoop)
+    ? Select the connector for mysql-corp: MySQL (supported by StrongLoop)
     ```
 
     ```shell
     $ slc loopback:datasource
     ? Enter the data-source name: mysql-corp
-    ? Select the connector for mysql: MySQL (supported by StrongLoop)
+    ? Select the connector for mysql-corp: MySQL (supported by StrongLoop)
     ```
 
-    Follow the prompts to name the datasource and select the connector to use.
-    This adds the new data source to `datasources.json`
+    Follow the prompts to fill in the connector configurations.
+    This adds the new data source to `server/datasources.json` which can be later used to update the config if needed.
```

Comment:
We use terminology "data source properties" not "config/configuration" or "settings".  Technically, they are for the data source (i.e. this instance), not the connector (the `loopback-connector-*` package).

> Follow the prompts to fill in the connector configurations.

Change to:
"Respond to the prompts to set data source properties such as database credentials."

> ... which can be later used to update the config if needed

Change to:

"To change the data source properties, edit this file."